Mathlib/Order/Comparable.lean

```diff
@@ -0,0 +1,289 @@
+/-
+Copyright (c) 2025 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import Mathlib.Order.Antisymmetrization
+
+/-!
+# Comparability and incomparability relations
+
+Two values in a preorder are said to be comparable whenever `a ≤ b` or `b ≤ a`. We define both the
+comparability and incomparability relations.
```

Comment:
Let's add a comment here saying that CompRel is always true in a linear order and IncompRel is always false; as well as a (maybe simp) lemma proving this. 

Let's also add a TODO to make a linear order from a partial order where CompRel always holds (or you can do it in this PR, that's up to you).